cfripper/model/utils.py

```diff
@@ -101,3 +104,21 @@ def get_managed_policy(managed_policy_arn):
             }
         )
     return None
+
+
+def get_account_id_from_arn(arn) -> Optional[str]:
+    match = REGEX_ARN.match(arn)
+    if match:
+        return match.group(3)
+
+
+def get_account_id_from_iam_arn(arn) -> Optional[str]:
```

Comment:
This function isn't used 👀

Reply:
True, now the get_account_id_from_arn is not used, but it's ok. It's useful anyway.